src/util/ColorsUtil.js

```diff
@@ -0,0 +1,67 @@
+/*!
+ * Copyright (c) 2018-Present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+define(['util/Enums'], function (Enums) {
+  var fn = {};
+
+  var validate = function (colors) {
+    if (!colors.brand.match(/^#[0-9A-Fa-f]{6}$/)) {
```

Comment:
shall we follow same pattern like [this](https://github.com/okta/okta-signin-widget/blob/master/src/models/Settings.js#L320-L322) to fail earlier?